src/app/auth/authentication.service.ts

```diff
@@ -100,7 +100,9 @@ export class AuthenticationService {
       let refreshInMs = Math.round(refreshInSeconds * .9) * 1000;
       console.log('Refreshing token in: ' + refreshInMs + ' milliseconds.');
       this.refreshInterval = refreshInMs;
-      if (process.env.ENV !== 'inmemory') {
+      // Disable token refresh timeout when LOGIN_CLIENT_NO_REFRESH is set
+      // This is necessary to run fabric8-planner end-to-end tests
+      if ( ! (process.env.ENV === 'inmemory' || process.env.ENV === 'LOGIN_CLIENT_NO_REFRESH')) {
```

Comment:
rather than use various values for the `ENV`, could we keep it to the normal list of envs like `development`, `production`, `tests` (and `inmemory` since it is already used) and use different environment variable to toggle features. This allow us to toggle each feature for all `non-production` environments. 

```typescript
type TestFlag = 'inmemory' | ''LOGIN_CLIENT_NO_REFRESH' 

function test_flag_enabled(flag: Flag): bool {
  const ENV  = process.env.ENV; 
  if ( ENV === 'production') {
    return false
  }

 // NOTE: special case preserved for backward compatibility reason; see planner runtime webpack.dev.js 
 if ( flag === 'inmemory && ENV === 'inmemory) {
   return true
 }
 // support export ENV_VAR=true or ENV_VAR=1

 let value: string = process.env[flag] || ""
 return value === "true" || value === "1"
}

// use the above to toggle feature like ... 
if ( !test_flag_enabled('inmemory) && !test_flag_enabled('LOGIN_CLIENT_NO_REFRESH')) {
  // setTimeout .. 
}
```